packages/cozy-dataproxy-lib/src/dataproxy/DataProxyProvider.jsx

```diff
@@ -61,18 +63,59 @@ export const DataProxyProvider = React.memo(({ children, options = {} }) => {
       } catch (error) {
         setDataProxyServicesAvailable(false)
         log.error(
-          'Error whild initializing Search intent, dataproxy features will be disabled',
+          'Error while initializing Search intent, dataproxy features will be disabled',
+          error
+        )
+      }
+    }
+
+    const initFlagship = async () => {
+      try {
+        if (!webviewIntent) {
+          return
+        }
+
+        if (!flag('cozy.search.enabled')) {
```

Comment:
nit: Maybe the flagship flag condition could be centralized with the iframe flag condition ?

Reply:
Fixed [here](https://github.com/cozy/cozy-libs/compare/42705b3dd71aee1e46b39aab17f937d27f8869a4..a1afc01affa8cb1dd888bba9a989d324b11130b4)